recipe/meta.yaml

```diff
@@ -39,6 +40,9 @@ requirements:
     - fsspec
     - s3fs
     - gcsfs
+    - esmf * nompi*
```

Comment:
```suggestion
    - esmf * nompi*   # [nompi == 'nompi']
```

Reply:
Actually for esmf we always want the nompi build. This is basically because esmf is being used per-process in ufs2arco, and we don't want its operations to get in the way of ufs2arco's mpi usage.